src/App.js

```diff
@@ -1,8 +1,61 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function App() {
+  const [candidates, setCandidates] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  async function getVotes() {
+    try {
+      if(!loading) setLoading(true);
```

Comment:
```suggestion
      setLoading(true);
```

if 문이 굳이 있을 필욘 없을 것 같아요